android/build.gradle

```diff
@@ -91,9 +99,12 @@ android {
             "**/libreact_nativemodule_core.so",
             "**/libturbomodulejsijni.so",
             "**/MANIFEST.MF",
-            ""
     ]
-    doNotStrip '**/*.so'
+    // Should prevent clashes with other libraries that use OpenSSL
+    pickFirst '**/x86/libcrypto.so'
```

Comment:
Should prevent conflicts when other libraries also generate `libcrypto.so` like flipper or `op-sqlcipher`

Reply:
Yea I mean it's kinda unsafe here but that's the developers fault if he includes two libs with different major versions imo.